src/sentry/utils/integrationdocs.py

```diff
@@ -122,9 +123,14 @@ def sync_docs(quiet=False):
 
     dump_doc("_platforms", {"platforms": platform_list})
 
+    executor = multiprocessing.dummy.Pool(min(32, multiprocessing.cpu_count() * 5))
```

Comment:
Is this really intentional to use 32 processes min? That seems like a lot.

Reply:
Oh wait, TIL that `multiprocessing.dummy` is a wrapper aroudn the threading API.

Wouldn't you gain more benefits from using real multiprocessing? Why was it decided to do it this way? Depednign on what the bottleneck is, threading might not help as much.

Reply:
https://github.com/getsentry/sentry/pull/19014#discussion_r430560930

Reply:
I didn't actually do any serious introspection by the way, just watched CPU usage barely go up as I turned up the threads. So I assume it's I/O bound.

Reply:
@mattrobenolt, I took the number from here: https://docs.python.org/3/library/concurrent.futures.html#threadpoolexecutor

>Changed in version 3.8: Default value of max_workers is changed to min(32, os.cpu_count() + 4). This default value preserves at least 5 workers for I/O bound tasks. It utilizes at most 32 CPU cores for CPU bound tasks which release the GIL. And it avoids using very large resources implicitly on many-core machines.

It is number of threads and the reason to chose threads over processes was because this function is mostly I/O bound on network and GIL/CPU is not a limiting factor here.